src/usdb_syncer/gui/forms/SettingsDialog.ui

```diff
@@ -124,35 +124,16 @@
         <bool>true</bool>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
-        <item row="0" column="1">
-         <widget class="QSpinBox" name="cover_max_size">
-          <property name="sizePolicy">
-           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
-            <horstretch>0</horstretch>
-            <verstretch>0</verstretch>
-           </sizepolicy>
-          </property>
-          <property name="toolTip">
-           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Images larger than this will be scaled to the set value. Use 0 to disable.&lt;/p&gt;&lt;p&gt;Some USDX variants have problems displaying covers larger than 1920x1920 pixels.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>

```

Comment:
This removed the tooltip.

Reply:
Good catch, that was not removed on purpose.